easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
well, except that #1973 deals with icc/ifort 2016.0.109, while here you have 2016.1.150... The bump in the compiler version should be reflected in the toolchain version, no?

Now both this PR and #1973 propose to use `2016.00`... We want to support installing both the 2016.0.109 and 2016.1.150 versions (for benchmarking/testing, for example).